Week 6 Exercise 4.3.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<!--Week 6 Assignment: Wioleta Takuska-->
+<html lang = "en">
+
+<head>
+    <title> Wioleta Takuska : Exercise 4.3 </title>
+
+    <meta charset= "utf-8" />
+ 
+</head>
+
+<body>
+        
+    <script type="text/javascript" src="Week 6 Exercise 4.3.js"></script>
```

Comment:
I'm honestly surprised this works with the spaces. In Linux space characters normally have to be "escaped", a backslash put in front of them, to let the parser/terminal know that it's not the end of the string.